databook/background/Two-photon-calcium-imaging.md

```diff
@@ -35,13 +39,13 @@ neural ensembles encode, process, and transmit information.
   circuits with cell-type precision.
 - **Temporal insights**: While the temporal resolution of calcium imaging
   doesn't match that of direct electrical recordings, indicators have become
-  faster over each generation. Even compartively "slow" indicators such as
+  faster over each generation. Even comparatively "slow" indicators such as
   GCaMP6s are still fast enough to detect when a group of action potentials
   occur, even if it is difficult to discern individual action potentials.
-- **Longitudinal recordings**: Because you can see the shape of individual
-  cells, their arrangement with respect to eachother, and other tissue featuers
-  like vasculature, experimenters can measure the activity of the same neurons
-  over days. This can provide insight into how neural activity changes over the
+- **Longitudinal recordings**: Calcium imaging allows experimenters to discern
```

Comment:
Let's try:

Because calcium imaging allows experimenters to see the shape of individual
  neurons, as well as their arrangement with respect to each other and other tissue features
  (like vasculature), experimenters can measure the activity of the same neurons
  over multiple days or weeks. This can provide insight into how neural activity changes over the course of learning.

Reply:
revision made